modules/identities/scripts/sign_in.ts

```diff
@@ -0,0 +1,46 @@
+import { ScriptContext } from "../module.gen.ts";
+import { IdentityDataInput, IdentityProviderInfo } from "../utils/types.ts";
+
+export interface Request {
+    info: IdentityProviderInfo;
+    uniqueData: IdentityDataInput;
+}
+
+export interface Response {
+    userToken: string;
+}
+
+export async function run(
+	ctx: ScriptContext,
+	req: Request,
+): Promise<Response> {
+    const key = req.info.identityType + ":" + req.info.identityId + ":" + JSON.stringify(req.uniqueData);
```

Comment:
the json here is not consistent. if you do need to do this (instead of rate limiting at the source provider), let's chat on impl details for this.

Reply:
Sounds good. It doesn't actually need to be rate-limited here.